components/Footer.tsx

```diff
@@ -1,5 +1,5 @@
 import React from "react"
-import styles from "@/styles/Home.module.css"
+import styles from "@/styles/Home.module.scss"
```

Comment:
May be overkill, but I think we should move the styles out of Home and into a Footer.module.scss. Ditto for the other Home components.